auto_nag/scripts/missing_beta_status.py

```diff
@@ -32,29 +32,28 @@ def get_autofix_change(self):
 
     def handle_bug(self, bug, data):
         bugid = str(bug["id"])
-        central = bug[self.status_central]
+        nightly = bug[self.status_nightly]
         release = bug[self.status_release]
         doc = self.get_documentation()
 
-        if central == release:
-            if release != "verified":
+        # If the two status are different, we don't know what to set.
+        # If this affected on nightly and release, beta will be affected.
+        # Otherwise we cannot say per say if beta should be the same as nightly
+        # and release.
+        if nightly == release:
```

Comment:
They don't have to be the same, we should also mark it as affected for beta when release is wontfix (which assumes "affected") and nightly is affected.

Reply:
Or, for example, release is disabled and nightly is affected

Reply:
> They don't have to be the same, we should also mark it as affected for beta when release is wontfix (which assumes "affected") and nightly is affected.

Yes, but still we do not know if it is wontfix or just affected. But I assume it is safe to mark the beta as affected even if it is wontfix.



> Or, for example, release is disabled and nightly is affected

If it is disabled in release, it could be disabled in beta as well and not affected (i.e., regression happened in the nightly).

Reply:
Yeah, we might be partially wrong in setting "affected", but we are on the safe side (the downside is just that release managers will have to make an explicit choice, which in the end is not even a downside :D).
This applies both to wontfix and disabled:
- if release is wontfix and nightly is affected, it's a bug where it was decided that there was not enough time or importance to fix it for release, but now we need to decide about beta;
- if release is disabled and nightly is affected, it means the bug affects both but it was decided to disable in release the whole feature which the bug affects -> we don't know if the feature is disabled in beta too, so we mark it as affected and let the release managers or developers mark it as disabled

Reply:
In 0cc3879a25b6c6e6235a78d9c892b5d04f3f0bce, anything is not `unaffected` or `?`, it is considered as affected status. If both release and nightly are affected (e.g., `wontfix` or `disabled`), beta will be marked as `affected`.